_includes/getting-started.html

```diff
@@ -87,54 +87,37 @@ <h3>Kubernetes User:</h3>
 
             <!-- Kubernetes - Helm -->
             <h4 style="font-size: 1.5em;">Using Helm</h4>
-            <p>Helm v3 - run the following:</p>
             <div id="start-helm-v3" class="highlight-code">
-              <p class="code-box" style="height: 90px; line-height: 30px;"><code style="position: absolute;"> $ git clone https://github.com/layer5io/meshery.git; cd meshery
-                $ kubectl create namespace meshery
-                $ helm install meshery --namespace meshery install/kubernetes/helm/meshery
-                </code></p>
+              <p class="code-box" style="height: 60px; line-height: 30px;"><code style="position: absolute;">$ kubectl create namespace meshery
+                $ helm install meshery --namespace meshery install/kubernetes/helm/meshery</code></p>
                 <!-- copy to clipboard -->
                 <a class="btn tooltip" style="position: absolute; top:0.5rem; right: 0.5rem; font-size: 22px;" data-clipboard-target="#start-helm-v3"
-                  data-clipboard-text="git clone https://github.com/layer5io/meshery.git; cd meshery
-kubectl create namespace meshery
+                  data-clipboard-text="kubectl create namespace meshery
 helm install meshery --namespace meshery install/kubernetes/helm/meshery" onmouseout="resetCopyText(this)">
                   <i class="far fa-copy"></i>
                   <span class="tooltiptext" style="font-size: 15px; width: 140px; height: 40px; padding: 0; line-height: 40px; top: 3rem; left: -20px;">Copy to clipboard</span>
                 </a>
               </div>
 
-            <p>Helm v2 - run the following:</p>
+            <!-- <p>Helm v2 - run the following:</p>
```

Comment:
We can remove the content with Helm v2

Reply:
Oh, very good. No need to keep this old code in comments. Ok. Makes sense.